MAINTAINERS.md

```diff
@@ -30,6 +30,7 @@ Everybody listed is a committer as per governance definition.
  * [Kornilios Kourtis] (Isovalent)
  * [Laurent Bernaille] (Datadog)
  * [Liz Rice] (Isovalent)
+ * [Lorenz Bauer](Isovalent)
```

Comment:
```suggestion
 * [Lorenz Bauer] (Isovalent)
```

Otherwise this entry would be rendered as a link to `Isovalent` instead of https://github.com/lmb